### PR TITLE
Development of the Nytime Clone ready for a Merge to Master

### PR DESCRIPTION
**Pull request by dannisonarias**
Thanks for reviewing our project
We followed the guidelines as closely as possible! 
all elements are aligned with floats.

### CODE REVIEW

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**:clap: Great job team, a few changes and we will be good to :airplane:**
1. Linter tests are failing. Ensure that you address all the linter errors.



2. The new york times logo in the nav bar should be centered.
Compare yours with this:
image


3. Wrap the main navigation links in `nav` tags.



4. The background image in the **Theory of Inflation section** does not display in the live demo. Check how the image is referenced in your css. Probably, you have an unnecessary preceding slash before the file address.

Kindly make these changes and submit another code review request.
**Happy Coding :smile:**

**Status: Changes requested**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+        <li><a id="footer-bottom-nav-a" href="#">Work with us</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Advertise</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">T Brand Studio</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Your Ad Choices</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Privacy</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Terms of Service</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Terms of Sale</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Site Map</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Help</a></li>
+        <li><a id="footer-bottom-nav-a" href="#">Subscriptions</a></li>
```
Comment (pelzolga123) [inline index.html]: There should be no ID selectors for multiple elements, ID should be unique for each element, better use class instead.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
* [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
##### Navbar part
* [ ] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
##### Bottom part
* [ ] Should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by addod19** [comment]
## Pull Request under ***review*** by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi Team ✋ 
Good job fixing the changes requested by previous reviewers.

Kindly fix linter errors and submit code review. Linters are still failing

slint

Kindly fix these and submit code review. 👍 
Happy coding 💻 

[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Comment by addod19** [comment]
Kindly change the hamburger to button....Currently it is image

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
# Status [APPROVED]
## Great job on working on all requirements :+1: 


Go ahead and merge to master.
Good luck in your next project Happy coding

**Pull request merged**